source/components/text-input/text-input.less

```diff
@@ -47,14 +51,6 @@ textarea {
     border-color: @gray-300;
     color: @gray-400;
     cursor: not-allowed;
+    resize: none;
   }
 }
-
-textarea[disabled] {
-  resize: none;
-}
-
-// TODO: move to appropriate file
-.hxFormGroup {
```

Comment:
Used for old demo. No longer needed. We can add back later once we create styles for form layouts.